pfcpiface/messages-conn.go

```diff
@@ -59,8 +59,10 @@ func (pConn *PFCPConn) handleHeartbeatRequest(msg message.Message) (message.Mess
 		return nil, errUnmarshal(errMsgUnexpectedType)
 	}
 
-	// reset heartbeat expiry timer
-	pConn.hbReset <- struct{}{}
+	if pConn.upf.enableHBTimer {
+		// reset heartbeat expiry timer
+		pConn.hbReset <- struct{}{}
```

Comment:
Should we make this write timeout based? The receiver side could still be slow and I don't think we want to wait too long in a message handler.

Reply:
yes, that's a good idea